src/routes/+layout.svelte

```diff
@@ -25,12 +27,13 @@
 		if (!($settingsStore.autoCheckForUpdates === false)) await checkForUpdates();
 	});
 
-	beforeUpdate(() => {
-		loadLocale('en');
-		setLocale('en');
-	});
-
 	onMount(() => {
+		if (!$settingsStore.userLanguage)
+			$settingsStore.userLanguage = detectLocale('en', ['en', 'es']) as Locales;
```

Comment:
`detectLocale` accepts a fallback language as first parameter, and a list of available locales as second parameter. [Ref](https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/detectors#detectors)